apt/risk/data_assessment/attack_strategy_utils.py

```diff
@@ -74,3 +87,91 @@ def find_knn(self, knn_learner: NearestNeighbors, query_samples: ArrayDataset, d
             else:
                 distances.append(dist_batch)
         return np.concatenate(distances)
+
+    @staticmethod
+    def _column_statistical_test(df1_column_samples, df2_column_samples, column, is_categorical, is_numeric, test_type,
+                                 alpha, differing_columns):
+        if is_categorical(column):
```

Comment:
Why is is_categorical passed as a method and is_numeric as a boolean? Why not make both booleans?